src/window/utils.rs

```diff
@@ -0,0 +1,83 @@
+use glutin::dpi::PhysicalSize;
+use serde::{Deserialize, Serialize};
+use std::ops::{Div, Mul};
+
+// Maybe this should be independent from serialization?
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Serialize, Deserialize)]
+pub struct Dimensions {
```

Comment:
Is window/utis good place for this file? maybe it should be `src/utils/dimensions`? I really like src/utils/.

Is Dimensions good name for this struct?

Reply:
I think it should be nice to move to `src/utils` as it could be useful out of window module and I like the existence of a `src/utils` also

Dimensions should be good (or maybe Dimension, just to avoid using plural as a type name?), sometimes I use Size to describe something like that also